enos/enos-scenario-agent.hcl

```diff
@@ -3,19 +3,83 @@
-    account. To verify that your account has agreed, sign-in to your AWS through Doormat,
-    and visit the following links to verify your subscription or subscribe:
-      arm64 AMI: https://aws.amazon.com/marketplace/server/procurement?productId=a516e959-df54-4035-bb1a-63599b7a6df9
-      amd64 AMI: https://aws.amazon.com/marketplace/server/procurement?productId=5535c495-72d4-4355-b169-54ffa874f849
+    # How to run this scenario
+
+    1. Install Enos (more info: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/)
+    
+    $ brew tap hashicorp/tap && brew update && brew install hashicorp/tap/enos
+
+    2. Authenticate to AWS with Doormat (more info: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/#authenticate-to-aws-with-doormat)
+    
+    $ doormat login && eval $(doormat aws -a <your_aws_account_name> export)
+
+    3. Set the following variables either as environment variables (`export ENOS_VAR_var_name=value`)
+    or in your 'enos-local.vars' file:
+
+    Required variables
+      - aws_ssh_private_key_path (more info about AWS SSH keypairs: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/#set-your-aws-key-pair-name-and-private-key)
+      - aws_ssh_keypair_name
+      - vault_build_date
+      - vault_product_version
+      - vault_revision
+
+    Optional variables
+      - aws_region (if different from the default value in enos-variables.hcl)
+      - consul_license_path (if using an ENT edition of Consul)
+      - distro_version_<distro> (if different from the default version for your target
+      distro; see list of supported distros and versions in enos-globals.hcl)
+      - vault_license_path (if using an ENT edition of Vault)
+    
+    4. Choose one of the following ways to get a Vault artifact and set the appropriate variables:
+
+      a. 'artifact_source:crt'
+      This will use a Vault artifact that you have already downloaded. Set the following variable:
+
+      ENOS_VAR_vault_artifact_path=path/to/existing-vault-artifact.[zip | rpm | deb]
+
+      b. 'artifact_source:local'
+      This will build a Vault .zip bundle from your local branch. Set the following variable:
+
+      ENOS_VAR_vault_artifact_path=path/to/where/vault-should-be-built.zip
+
+      c. 'artifact_source:artifactory'
+      This will download a Vault artifact of the specified version, edition, and revision SHA
+      from the 'stable' repo in Artifactory. Set the following variables:
+
+      export ENOS_VAR_artifactory_username=your-user
+      export ENOS_VAR_artifactory_token=your-token
+    
+    5. Choose the matrix variants you want to use, and launch the scenario with the appropriate
+    filter for those variants, e.g.:
+
+    $ enos scenario launch agent arch:amd64 artifact_source:crt artifact_type:bundle backend:consul config_mode:env consul_edition:ent consul_version:1.16.3 distro:ubuntu edition:ent.hsm.fips1402 seal:pkcs11
+
```

Comment:
IMO this should be a code block and maybe use line breaks to make it more readable:

```
enos scenario launch agent \
    arch:amd64 \
    artifact_source:crt \
    artifact_type:bundle \
    backend:consul \
    config_mode:env \
    consul_edition:ent \
    consul_version:1.16.3 \
    distro:ubuntu \
    edition:ent.hsm.fips1402 \
    seal:pkcs11
```